.github/workflows/benchmark.yml

```diff
@@ -68,7 +68,7 @@ jobs:
   tpcds-1g-gen:
     name: "Generate an input dataset for TPCDSQueryBenchmark with SF=1"
     if: contains(inputs.class, 'TPCDSQueryBenchmark') || contains(inputs.class, '*')
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
Can you manually run a `org.apache.spark.sql.execution.benchmark.TPCDSQueryBenchmark` to ensure that this modification is okay?

Reply:
Sure. Waiting for:

- jdk17: https://github.com/wayneguow/spark/actions/runs/13363949014
- jdk21: https://github.com/wayneguow/spark/actions/runs/13363952357